packages/governance/test/swingsetTests/committeeBinary/bootstrap.js

```diff
@@ -1,15 +1,15 @@
-import { E } from '@endo/eventual-send';
-import { Far } from '@endo/marshal';
-import { makeFakeBoard } from '@agoric/vats/tools/board-utils.js';
 import { makeMockChainStorageRoot } from '@agoric/internal/src/storage-test-utils.js';
 import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/marshal';
```

Comment:
While you're changing this, you can use the preferred form:
```suggestion
import { E, Far } from '@endo/far';
```

Reply:
There are over a hundred files with the two lines here. If `far` is preferred, I'll do that in a follow-up PR.

Reply:
https://github.com/Agoric/agoric-sdk/pull/8180